### PR TITLE
Update portal features from v502

### PR DESCRIPTION
**Pull request by sjpadgett**
- register improvements
- exclude in portal layouts

### CODE REVIEW

**Comment by sjpadgett** [comment]
@bradymiller If you get a chance will you give a once over. I'm tired and moving between v502 and v6 has me dizzy!!!
I still, down the road, want to review how we present profile inputs. We can prob trim some of the viewport footprint down some.

Now for register and profile edits any items not wanted can be excluded in the DEM layouts. umm, have I not said this like 10 times already!!!:) I'll still get asked though.:)
thanks

**Comment by sjpadgett** [comment]
why are we running v6 master PSR styling rules against v502 branch?

**Comment by sjpadgett** [comment]
@bradymiller I haven't heard any negatives concerning billing, dicom or portal for patch 4. it's been beat up pretty good so maybe time to release it.
Plus, I can't seem to remember what branch i'm working in any longer!

File: portal/account/account.lib.php [hunk 1]
```diff
@@ -200,8 +231,27 @@ function doCredentials($pid)
         $sent = 1;
     } else {
         $email_status = $mail->ErrorInfo;
-        error_log("EMAIL ERROR: " . errorLogEscape($email_status), 0);
-        return xlt("EMAIL ERROR") . ": " . errorLogEscape($email_status) . ': ' . xlt("Contact your provider administrator.");
+        $errorMsg = "EMAIL ERROR: " . errorLogEscape($email_status) . '<br />';
+        if ($newpd['id']) {
+            $errorMsg .= xlt("Your account has been successfully created however, we were unable to send the account information.");
+            $errorMsg .= "<br />" . xlt("Please contact your providers office with the following account information") . ":<br />";
+            $errorMsg1 = xlt("Account Id") . ": " . $uname . " " . xlt("MRN Reference") . ": " . $pid;
```
Comment (bradymiller) [inline portal/account/account.lib.php:238]: if this is going to html, then would place text() call around $uname and $pid

File: portal/account/account.php [hunk 2]
```diff
@@ -103,7 +99,8 @@
     unset($_SESSION['site_id']);
     unset($_SESSION['register']);
     echo 'gone';
-    OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy(); // I know, makes little sense.
+    OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy();
+// I know, makes little sense.
```
Comment (bradymiller) [inline portal/account/account.php:103]: i like this comment :)

**Comment by bradymiller** [comment]
hi @sjpadgett , looks good (only noted 2 minor escaping issues). i get confused sometimes too. For example, just the other day, I was proposing this as OpenEMR new official logo :)

tenor (39)

**Comment by bradymiller** [comment]
great news on patch 4. will shoot for releasing it by wednesday

2DV

**Pull request merged**